src/Symfony/Bridge/Doctrine/PropertyInfo/DoctrineExtractor.php

```diff
@@ -133,7 +133,7 @@ public function getType(string $class, string $property, array $context = []): ?
 
         // DBAL 4 has a special fallback strategy for BINGINT (int -> string)
         if (Types::BIGINT === $typeOfField && !method_exists(BigIntType::class, 'getName')) {
-            return Type::collection(Type::int(), Type::string());
+            return $nullable ? Type::union(Type::int(), Type::string()) : Type::nullable(Type::union(Type::int(), Type::string()));
```

Comment:
The ternary looks wrong to me. Why are you marking the type as nullable when `$nullable` is `false` ?

Reply:
Yep indeed, a little mistake of mine, fixed!